files/www/cgi-bin/status

```diff
@@ -392,15 +406,21 @@ if tspace < 3000 then
 else
     tspace = tspace .. " KB"
 end
-local rspace = (sysinfo.freeram + sysinfo.bufferram) / 1024
+local rspace = sysinfo.freeram / 1024
 if rspace < 500 then
     rspace = "<blink><b>" .. rspace .. " KB</b></blink>"
 else
     rspace = rspace .. " KB"
 end
+local mavail = get_memavail()
+if mavail < 500 then
```

Comment:
While extremely unlikely, get_memavail can return nil (it didnt find a value) and this comparison will error and stop the page being rendered. Need to handle that case or return a default value from get_memavail.

Reply:
Good catch, @aanon4 Tim.  THX